drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -2080,6 +2080,22 @@ matterManufacturer:
     vendorId: 0x139C
     productId: 0xEEE2
     deviceProfileName: switch-binary
+  - id: "5020/53249"
+    deviceLabel: Zemismart downlight
+    vendorId: 0x139C
+    productId: 0xD001
+    deviceProfileName: light-color-level
+  - id: "5020/53250"
+    deviceLabel: Zemismart bulb
+    vendorId: 0x139C
+    productId: 0xD002
+    deviceProfileName: light-color-level
+  - id: "5020/53251"
+    deviceLabel: Zemismart Led Light Strip Driver
```

Comment:
```suggestion
    deviceLabel: Zemismart LED Light Strip Driver
```
Should this be LED?

Reply:
This is what the Partner has provided, but I think we can make that improvement.